content/news/2018-11-14-community-fund-locks-in.md

```diff
@@ -0,0 +1,74 @@
+
+After this block, nodes running 4.3.x and below have an increasing risk of ending up on a fork from the main chain and forking will become unavoidable when the first payment request is processed.
+
+It is highly recommended to upgrade to version 4.4.0 as soon as possible to avoid any problems after the consensus change activates. After the soft fork activates, the upgrade is considered mandatory and all older versions of NavCoin Core become deprecated.
+
+
+
+[You can download the latest wallets here.](https://navcoin.org/en/wallets/)
+
+### Community Fund submissions
+
+Already there are interesting proposals appearing on the NavCollab site, Reddit, and #HumansOfNavCoin. If you have a great idea, but not all the skills to bring it to life - don’t let that stand in your way - just put your idea out there and don’t be shy about asking for help from other community members. Within this global community, there are remarkable people with a range of skills to call on. 
+
+Remember what you propose is entirely up to you - all you have to do is convince the community that your idea is worth voting for.
+
+Here are some easy ways to get started now: 
+
+* [Go to the NavCollab website](https://collab.navcoin.org/dashboard), create a profile, and create a Space where you can propose your idea.
+* Post an idea in [the Community Fund channel](https://discord.gg/y4Vu9jw) on Discord, or [in the NavCoin subreddit](https://www.reddit.com/r/NavCoin/).
+* Post your idea to Twitter and use the #HumansOfNavCoin hashtag to help it get seen. You can also [search #HumansOfNavCoin on Twitter](https://twitter.com/search?f=tweets&vertical=default&q=%23HumansOfNavCoin) to see what others have suggested.
+
+### Proposal submission tools
+
+Once you’re ready there are two ways to submit your proposal to the blockchain to be voted on by NavCoin stakers:
+
+* [Sakdeniz’ NEXT wallet](http://next.navcommunity.net/) includes a built-in Community Fund proposal form. 
+* Proposal generator + NavCoin Core wallet. Go to the [Community Fund page](https://navcoin.org/en/community-fund/) and scroll down to the ‘How to Create A Proposal’ section in Step 4 to a form that generates the code to paste in your NavCoin core wallet.
+
+### NavCoin lottery proposal on Reddit.
+
+Nav_Whale posted an [interesting idea for a NavCoin Lottery](https://www.reddit.com/r/NavCoin/comments/9t3f4x/navcoin_lottery_as_a_proposal_for_community_fund/) that would reward stakers by putting them into a monthly draw to win NAV. Check out the comments below this proposal to see what other community members are feeling, and add your thoughts. 
+
+### Marcus goes full-time + summer interns
+
+Marcus, who was involved in testing the Community Fund, has started working full-time with the NavCoin Core devs during his summer uni break. 
```

Comment:
summer break = winter for most of the world, took me a moment

Reply:
@kierenhyl maybe just say "university break" instead of summer uni break.